crates/biome_cli/tests/commands/ci.rs

```diff
@@ -694,15 +702,8 @@ something( )
 
     assert!(result.is_err(), "run_cli returned {result:?}");
 
-    let mut file = fs
-        .open(file_path)
-        .expect("ci target file was removed by the CLI");
-
-    let mut content = String::new();
-    file.read_to_string(&mut content)
-        .expect("failed to read file from memory FS");
+    assert_file_contents(&fs, file_path, expect);
```

Comment:
Here you went beyond the task; fantastic work! Not sure why the old code is like this... we completely forgot to do the content assertion!